### PR TITLE
Ticket21315

### PR DESCRIPTION
**Pull request by cohosh**

### CODE REVIEW

**Comment by cohosh** [comment]
Looks like Travis doesn't like the convey package. The branch is passing CI, even though this PR isn't.

File: broker/broker.go [hunk 1]
```diff
@@ -289,7 +292,7 @@ func main() {
 		metricsFile = os.Stdout
 	}
 
-	metricsLogger := log.New(metricsFile, "", log.LstdFlags|log.LUTC)
+	metricsLogger := log.New(metricsFile, "", 0)
```
Comment (NullHypothesis) [inline broker/broker.go]: Why are we getting rid of the flags here? I'm not saying we shouldn't; I'm just curious.

Comment (cohosh) [reply broker/broker.go]: The flags for loggers are mostly about formatting timestamps and prefixes for log messages. We don't want any prefixes for the log file, only the information in the specification.

File: broker/metrics.go [hunk 2]
```diff
@@ -82,6 +82,10 @@ func (m *Metrics) UpdateCountryStats(addr string) {
 	var country string
 	var ok bool
 
+	if m.countryStats.addrs[addr] {
```
Comment (NullHypothesis) [inline broker/metrics.go]: Go suggests the use of two-value assignments to test for the existence of a key, but I think relying on the zero value of a `bool` should also be fine: https://blog.golang.org/go-maps-in-action#TOC_3.

**Pull request merged**